nautobot/ui/src/utils/store.js

```diff
@@ -36,6 +36,12 @@ const appContextSlice = createSlice({
 const initialAppState = {
     currentContext: "Inventory", // TODO: What is the context for the homepage?
     routeToContext: {},
+    user: {},
+    logged_in: false,
+    auth: {
```

Comment:
should `user` and `logged_in` be subitems under `auth` to keep things organized long-term?